source/manifest.json

```diff
@@ -1,4 +1,10 @@
 {
+  "applications": {
+    "gecko": {
+      "id": "theoldreader@knyar",
+      "strict_min_version": "53.0"
+    }
+  },
```

Comment:
It's necessary for [`storage.sync`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage/sync). If it's the only difference, that could be easily solved with a simple build script and separate manifests. Or we can drop sync support for Firefox, what sound better for you.